LiteCore/Database/SequenceTracker.cc

```diff
@@ -83,18 +98,18 @@ namespace litecore {
         // Placeholder entry (when docID == nullslice):
         CollectionChangeNotifier* const   databaseObserver {nullptr};
 
-        Entry(const alloc_slice &d, alloc_slice r, sequence_t s, uint32_t bs, RevisionFlags flags)
-        :docID(d), revID(r), sequence(s), bodySize(bs), flags(flags), idle(false), external(false)
+        Entry(CollectionSpec c, const alloc_slice &d, alloc_slice r, sequence_t s, uint32_t bs, RevisionFlags flags)
+        :collection(c), docID(d), revID(r), sequence(s), bodySize(bs), flags(flags), idle(false), external(false)
         {
             DebugAssert(docID != nullslice);
         }
 
         explicit Entry(const alloc_slice &d)
-        :Entry(d, nullslice, 0_seq, 0, {})
+        :Entry(kC4DefaultCollectionSpec, d, nullslice, 0_seq, 0, {})
```

Comment:
SequenceTracker is owned by CollectionImpl. I feel we don't need extra collection info in the Entry, if we store the collectionSpec in the SequenceTracker.

Reply:
Is there a path up to the callback in that case?  Entries are used to generate document ended callback events.